### PR TITLE
Adds cross-linking to Akka and Java SE javadocs

### PR DESCRIPTION
**Pull request by ignasi35**
related to https://github.com/playframework/playframework/pull/8317
related to https://github.com/lightbend/platform-reconciliation/issues/33
related to https://github.com/lagom/lagom/pull/1283

### CODE REVIEW

**Comment by mkurz** [comment]
Seems like https://github.com/lightbend/platform-reconciliation/issues/33 is meant for internal usage only :stuck_out_tongue:

File: framework/project/Docs.scala [hunk 1]
```diff
@@ -146,6 +146,13 @@ object Docs {
 
       val javadocOptions = Seq(
         "-windowtitle", label,
+        // Adding a user agent when we run `javadoc` is necessary to create link docs
+        // with Akka (at least, maybe play too) because doc.akka.io is served by Cloudflare
+        // which blocks requests without a User-Agent header.
+        "-J-Dhttp.agent=Lagom-Unidoc-Javadoc",
```
Comment (2m) [inline framework/project/Docs.scala]: `Play-Unidoc-Javadoc` maybe?

Comment (ignasi35) [reply framework/project/Docs.scala]: "Kids, don't drink and copy/paste"

**Comment by ignasi35** [comment]
hi @marcospereira, I was reviewing this at [play's site](https://www.playframework.com/documentation/2.6.x/api/java/play/components/AkkaComponents.html) and I think I'm missing a detail or this PR didn't work. :-(

**Pull request merged**